agr_literature_service/api/schemas/__init__.py

```diff
@@ -81,6 +81,14 @@
     WorkflowTagSchemaRelated,
     WorkflowTagSchemaUpdate)
 
+from agr_literature_service.api.schemas.topic_entity_tag_schemas import (
+    TopicEntityTagPropSchemaCreate,
+    TopicEntityTagPropSchemaShow,
```

Comment:
Do we need a TopicEntityTagPropSchemaUpdate ?

Reply:
I will add a test to see if I can update. Yes, we probably need an update for the props. Also a delete.

Reply:
Nope not really needed. We can update via the TopicEntityTag. I added an example of this in test_018...